scalafmt-core/shared/src/main/scala/org/scalafmt/internal/Router.scala

```diff
@@ -27,7 +27,7 @@ import scala.meta.{
 }
 
 // Too many to import individually.
-import scala.meta.tokens.Token._
+import scala.meta.tokens.Token.{Space => _, _}
```

Comment:
I would be inclined to use 
```scala
 import scala.meta.tokens.{Token => T}
```
and prefix `T.Space` everywhere. Unimporting `Space` isn't a great long-term solution

Reply:
Does that mean we have to write now

```
import scala.meta.tokens.{Token => T}

a.is[T.RightArrow]
b.is[T.Import]
c.is[T.Space]
```

for __every__ `Token`?

Or do you want something like
```
import scala.meta.tokens.Token.{Space => _, _}
import scala.meta.tokens.{Token => T}

a.is[RightArrow]
b.is[T.Space]
```
?

Reply:
The current diff is fine, but if I were writing everything from scratch I would probably do
```scala
import scala.meta.tokens.{Token => T}

a.is[T.RightArrow]
b.is[T.Import]
c.is[T.Space]
```